styles/image.css

```diff
@@ -0,0 +1,10 @@
+.image-responsive {
+  max-width: 100%;
+  height: auto;
+}
```

Comment:
for responsive image you can go with max-inline-size, block size property

Reply:
Can You please tell me more as i am not getting you

Reply:
Done!!